sync/phabricator/listen.py

```diff
@@ -0,0 +1,121 @@
+import time
+import re
+from phabricator import Phabricator
+
+from .. import log
+from ..tasks import handle
+
+
+logger = log.get_logger(__name__)
+
+
+class PhabEventListener:
+
+    blacklist = ["added inline comments to D",
+                 "added a comment to D",
+                 "added a reviewer for D",
+                 "removed a reviewer for D",
+                 "requested review of D",
+                 "requested changes to D",
+                 "added a subscriber to D",
+                 "added a project to D",
+                 "committed rMOZILLA",
+                 "committed rVCT",
+                 "committed rCIADMIN",
+                 "edited reviewers for D",
+                 "updated the summary of D",  # Maybe useful to upstream info?
+                 "accepted D",  # Maybe useful to upstream info?
+                 "retitled D",  # Maybe useful to upstream info?
+                 "blocking reviewer(s) for D",
+                 "planned changes to D",
+                 "resigned from D"]
+
+    event_mapping = {
+        "updated the diff for D": "commit",
+        "created D": "commit",
+        "closed D": "closed",
+        "abandoned D": "closed",  # Bit unsure about this one
```

Comment:
I think the difference is that abandoned means that this isn't going to land, whereas closed happens when it does land? So I guess we want to treat those cases separately.

Reply:
That makes sense, I agree we should probably be handling that differently